pkg/metrics/aggregator.go

```diff
@@ -9,13 +9,14 @@ import (
 )
 
 const (
-	providerLabel       = "provider"
-	osdExporterValue    = "osd_exporter"
-	proxyHTTPLabel      = "http"
-	proxyHTTPSLabel     = "https"
-	proxyCALabel        = "trusted_ca"
-	proxyCASubjectLabel = "subject"
-	clusterIDLabel      = "_id"
+	providerLabel         = "provider"
+	osdExporterValue      = "osd_exporter"
+	proxyHTTPLabel        = "http"
+	proxyHTTPSLabel       = "https"
+	proxyCALabel          = "trusted_ca"
+	proxyCASubjectLabel   = "subject"
+	clusterIDLabel        = "_id"
+	cpmsInstanceTypeLabel = "label_beta_kubernetes_io_instance_type"
```

Comment:
I'm assuming this is referring to the well-known k8s label - in which case I'd recommend we don't use a deprecated label as reference: https://kubernetes.io/docs/reference/labels-annotations-taints/#beta-kubernetes-io-instance-type-deprecated - this is replaced by `node.kubernetes.io/instance-type`.